sessionctx/stmtctx/stmtctx.go

```diff
@@ -186,6 +186,10 @@ type StatementContext struct {
 	OptimInfo map[int]string
 	// InVerboseExplain indicates the statement is "explain format='verbose' ...".
 	InVerboseExplain bool
+
+	// TableToColumnMaps is a map from tableID to a series of maps. The maps are needed when checking data consistency.
+	// Save them here to reduce redundant computations.
+	TableToColumnMaps map[int64]interface{}
```

Comment:
Is it possible that resuing the same map for the whole transaction since a transaction will use the same schema version fetched using `start_ts`.

Reply:
Makes sense.